pkg/geo/geogen/geogen_test.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+// Package geogen provides utilities for generating various geospatial types.
+package geogen
+
+import (
+	"math/rand"
+	"strconv"
+	"testing"
+
+	"github.com/cockroachdb/cockroach/pkg/geo/geopb"
+	"github.com/stretchr/testify/require"
+	"github.com/twpayne/go-geom"
+)
+
+const numRuns = 25
+
+func TestRandomValidLinearRingCoords(t *testing.T) {
+	rng := rand.New(rand.NewSource(0))
```

Comment:
following up from standup, I think you want to be using `randutil.NewPsuedoRand` so that your seed is different each time. Otherwise, this function does the same thing on every invocation.

Reply:
I don't mind too much this is doing the same thing (it's a test that makes sure it generates something sane)
as long as the real random does the real thing.
that said, sure let's do it